components/About/AboutDetails.tsx

```diff
@@ -0,0 +1,70 @@
+import { Header } from "../Header/Header";
+import Image from "next/image";
+import {
+  WhatsappShareButton,
+  WhatsappIcon,
+  FacebookIcon,
+  FacebookShareButton,
+  PinterestShareButton,
+  PinterestIcon,
+} from "next-share";
+import {
+  AboutText,
+  Adress,
+  ContentContainer,
+  Images,
+  OwnerImage,
+  Phone,
+  SocialMedia,
+  StoreImage,
+} from "./AboutDetails.styled";
+import { ABOUT_TEXT, ABOUT_TITLE, ADRESS, PHONE_NUMBER } from "./constants";
+
+export const AboutDetails = () => {
+  return (
+    <>
+      <Header />
```

Comment:
if you want this reader on all pages, you can move this to `_app` or create a [layout component](https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#layout-pattern) to handle header/footer